appconfig/settings/segmentedchainedtickerplant.q

```diff
@@ -0,0 +1,37 @@
+\d .
+
+createlogs:0b;             	/- create a log file
+
+\d .sctp
+
+chainedtp:1b;                  /- differentiates between stp and sctp codebases
+tickerplantname:`stp1;         /- list of tickerplant names to try and make a connection to
+subscribeto:`;                	/- list of tables to subscribe for
+subscribesyms:`;              	/- list of syms to subscription to
+replay:0b;                    	/- replay the tickerplant log file
+schema:1b;                    	/- retrieve schema from tickerplant
```

Comment:
Can you make sure all the comments are squared off, and use the same approach (either // or /-)
Use spaces rather than tabs to square them off (the size of a tab is OS dependent)

Reply:
strange, it was lined up in vs code, will make this change though